python_src/src/lamp_py/postgres/seed_metadata.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import logging
+import sys
+from typing import List
+
+import sqlalchemy as sa
+
+from lamp_py.runtime_utils.import_env import load_environment
+
+from .postgres_schema import (
+    MetadataLog,
+    SqlBase,
+    VehicleEventMetrics,
+    VehicleEvents,
+    VehicleTrips,
+)
+from .postgres_utils import DatabaseManager
+
+logging.getLogger().setLevel("INFO")
+
+DESCRIPTION = """Interact with Performance Manager RDS for Testing"""
+
+
+def parse_args(args: List[str]) -> argparse.Namespace:
+    """parse args for running this entrypoint script"""
+    parser = argparse.ArgumentParser(description=DESCRIPTION)
+
+    parser.add_argument(
+        "--env-file",
+        dest="env_file",
+        help="environment file containing rds connection parameters",
+        required=True,
+    )
```

Comment:
i kind of want to make this a flag we pass into the other argparses and get rid of our code that tries to find it that we only use when running it on the command line. it feels cleaner to me.